internal/crypto/internal_ca.go

```diff
@@ -108,112 +105,6 @@ func makeSelfSignedCAConfig(subject pkix.Name, caLifetime time.Duration) (*oscry
 	return caConfig, nil
 }
 
-type CertificateExtensionFunc func(*x509.Certificate) error
-
-func (ca *CA) MakeServerCert(hostnames []string, expiryDays int, fns ...CertificateExtensionFunc) (*TLSCertificateConfig, error) {
```

Comment:
This function moved from `internal/crypto/cert.go` to `internal/crypto/internal_ca.go` in the first commit but moved back to the original file in the second commit.

Reply:
It think it is removed later on. It is not portable to plugins. At all.